miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -19,11 +19,15 @@ const.ERR_ACCT_MUST_BE_A_FAUCET=0x00020001
 # EVENTS
 # =================================================================================================
 
+# Event emitted to signal that an asset is going to be added to the account vault.
+const.ACCOUNT_VAULT_BEFORE_ADD_ASSET_EVENT=131072
 # Event emitted to signal that an asset is being added to the account vault.
-const.ACCOUNT_VAULT_ADD_ASSET_EVENT=131072
+const.ACCOUNT_VAULT_AFTER_ADD_ASSET_EVENT=131073
```

Comment:
I would update the comments as follows:

- "Event emitted before an asset is added to the account vault."
- "Event emitted after an asset is added to the account vault."

This comment also applies to other event comments.